docs/api/synopsis.md

```diff
@@ -1,18 +1,22 @@
 # Synopsis
 
-All of [node.js's built-in modules](http://nodejs.org/api/) are available in
-Electron, and third-party node modules are fully supported too (including the
-[native modules](../tutorial/using-native-node-modules.md)).
+All of [Node.js's built-in modules](http://nodejs.org/api/) are available in
+Electron and third-party node modules also fully supported as well (including
+the [native modules](../tutorial/using-native-node-modules.md)).
 
 Electron also provides some extra built-in modules for developing native
 desktop applications. Some modules are only available on the main process, some
-are only available on the renderer process, and some can be used on both processes.
-The basic rule is: if a module is GUI or low-level system related, then it should
-be only available on the main process. You need to be familiar with the concept of
+are only available on the renderer process (web page), and some can be used on
```

Comment:
Yeah I've gone back and forth on if it should be "on __ process" or "in __ process". I think "in". I'll make the changes here and do another PR to make sure all of them are that way across the docs.